### ISSUE TITLE
Alciphon - a database on insects of Serbia

### ISSUE THREAD

**Issue by gbif-portal** [gbif/data-mobilization#436]
## Alciphon - a database on insects of Serbia

Dataset link: https://alciphron.habiprot.org.rs/

Region: Republic of Serbia

Taxon: insects

Type: occurrence

Why is this important: Has also media files

Priority: medium

License: Unspecified

Comments: https://habiprot.org.rs/en/alciphron-news/

Dataholders contact information: habiprot@sbb.rs

**Comment by spalp** [comment]
Contacted the deputy and the chairman of the governing board of the [HabiProt](https://habiprot.org.rs/en) association.

**Comment by spalp** [comment]
Replied that they have been already contacted and  by by Svetlana Miteva and agrred to be moblised by a project of [The Habitat Foundation](https://thehabitatfoundation.org/)
@CecSve Should we close this issue at this point? Or does one close an issue here, only when the dataset has been already regsitered at GBIF?

**Comment by spalp** [comment]
I added data-mobilizatoin to indicate it is in the process of being moblised. Is that the right thing to do?

**Comment by CecSve** [comment]
Usually, we close once the dataset has been mobilized or if it will not be mobilized. We can use this issue to keep track of the process (publisher registration etc.). Does it mean that you are not assisting them in the process?


> I added data-mobilizatoin to indicate it is in the process of being moblised. Is that the right thing to do?

There is no description of the label, so I am not sure. I think it makes good sense since I don't think we have any label that indicates ongoing mobilization efforts. I will update the description so it fits.

**Comment by spalp** [comment]
> Usually, we close once the dataset has been mobilized or if it will not be mobilized. We can use this issue to keep track of the process (publisher registration etc.). Does it mean that you are not assisting them in the process?
> 
Yes, I am not assisting them. I have unassigned from this issue to be clear that it is not me who is supporting them.

> > I added data-mobilizatoin to indicate it is in the process of being moblised. Is that the right thing to do?
> 
> There is no description of the label, so I am not sure. I think it makes good sense since I don't think we have any label that indicates ongoing mobilization efforts. I will update the description so it fits.

Yes, thanks. That woudl definitely clarify things.

**Comment by CecSve** [comment]
> > Usually, we close once the dataset has been mobilized or if it will not be mobilized. We can use this issue to keep track of the process (publisher registration etc.). Does it mean that you are not assisting them in the process?
> 
> Yes, I am not assisting them. I have unassigned from this issue to be clear that it is not me who is supporting them.
> 

Thanks- it is always a bit tricky to keep track of progress when it is out of 'our' hands. What I have previously done, is to look up the publisher after a while and see if they are on GBIF.org. But if you are somehow kept in the loop, feel free to update the issue if you are aware of any progress.
